From a8f71d516c58f6e9f0ef19409c78e8c45826b0b6 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey.doherty@gmail.com>
Date: Fri, 4 Jul 2014 18:26:54 +0100
Subject: [PATCH] configure: Disable format error

pygobject fails to build due to its own strict compiler flags.
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index bda8eef..14a731d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -226,7 +226,6 @@ if test "x$GCC" = "xyes"; then
   JH_ADD_CFLAG([-Werror=pointer-arith])
   JH_ADD_CFLAG([-Werror=format-security])
   JH_ADD_CFLAG([-Werror=missing-include-dirs])
-  JH_ADD_CFLAG([-Werror=format])
   JH_ADD_CFLAG([-fno-strict-aliasing])
 
   case $host_os in
-- 
1.9.1

